js.js

```diff
@@ -22,6 +25,7 @@ var hasBeenClickedPause = false;
 var Seconds = document.querySelector("#Seconds");
 var Minutes = document.querySelector("#Minutes");
 // var div10 = (document.querySelector("div10").style.display = "none");
+//div10Spinner.display = "none";
```

Comment:
I didn't use the div10 element, instead directly accessed the lds-ring class using document.querySelector(".lds-ring").style.display, later on in the code
(ignore the div10Spinner, it's unused)